sympy/concrete/summations.py

```diff
@@ -574,13 +576,16 @@ def eval_sum(f, limits):
 
 
 def eval_sum_direct(expr, limits):
+    from sympy.core import Add
```

Comment:
For info on the rationale: https://github.com/sympy/sympy/issues/7893 & https://github.com/sympy/sympy/issues/7124.